annif/openapi/annif.yaml

```diff
@@ -96,23 +92,68 @@ paths:
               schema:
                 $ref: '#/components/schemas/SuggestionResultList'
         "400":
-          description: Bad Request
-          content:
-            application/problem+json:
-              schema:
-                $ref: '#/components/schemas/Problem'
+          $ref: '#/components/responses/BadRequest'
         "404":
-          description: Project not found
-          content:
-            application/problem+json:
-              schema:
-                $ref: '#/components/schemas/Problem'
+          $ref: '#/components/responses/NotFound'
         "503":
-          description: Service Unavailable
+          $ref: '#/components/responses/ServiceUnavailable'
+  /projects/{project_id}/suggest-batch:
+    post:
+      tags:
+      - Automatic subject indexing
+      summary: suggest subjects for given documents using batched backend operations
```

Comment:
I think this is going into too much implementation detail. How about

    suggest subjects for the given batch of up to 32 documents